README.md

```diff
@@ -100,15 +100,15 @@ fn try_xdp_firewall(ctx: XdpContext) -> Result<u32, ()> {
 When naming stucts and fields, we are trying to stick to the following
 principles:
 
-* Use `CamelCase`, even for names which normally would be all uppercase
+- Use `CamelCase`, even for names which normally would be all uppercase
```

Comment:
This reformatting seems unrelated to the change you are making. I am not a maintainer though so can't say if that should be excluded from the PR.

Reply:
it's my editor doing auto formatting :sweat_smile: , I will revert it

Reply:
Whoops, I noticed the comment after merging.

I was thinking before if I should point it out. Overall I'm fine with this re-format, but I agree it would be better to ship it separately. If anyone else is interested (@pythops ?), I would love to see a PR which makes the formatting more consistent.